charts/kube-acp-stack/README.md

```diff
@@ -0,0 +1,93 @@
+
+See the [acp](https://github.com/cloudentity/acp-helm-charts/tree/master/charts/acp) README for details about components, capabilities of OAuth/OIDC server with an advanced authorization, consent management, and developer enablement.
+
+## Prerequisites
+
+- Kubernetes 1.16+
+- Helm 3+
+
+## Get Repo Info
+
+```console
+helm repo add acp https://artifactory.cloudentity.com/acp-helm-charts
+helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+```console
+# Helm
+$ helm install [RELEASE_NAME] acp/kube-acp-stack
+```
+
+_See [configuration](#configuration) below._
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Dependencies
+
+By default this chart installs additional, dependent charts:
+
+- [acp/acp](https://github.com/cloudentity/acp-helm-charts/tree/master/charts/acp)
+- [cockroachdb/cockroachdb](https://github.com/cockroachdb/helm-charts/tree/master/cockroachdb)
+- [hazelcast/hazelcast](https://github.com/hazelcast/charts/tree/master/stable/hazelcast)
+
+To disable dependencies during installation, see [multiple releases](#multiple-releases) below.
+
+_See [helm dependency](https://helm.sh/docs/helm/helm_dependency/) for command documentation._
+
+## Uninstall Chart
+
+```console
+# Helm
+$ helm uninstall [RELEASE_NAME]
+```
+
+This removes all the Kubernetes components associated with the chart and deletes the release.
+
+_See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
+
+
+## Upgrading Chart
+
+```console
+# Helm
+$ helm upgrade [RELEASE_NAME] acp/kube-acp-stack
+```
+
+_See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
+
+### Upgrading an existing Release to a new major version
+
+A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
+
+## Configuration
+
+See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments:
+
+```console
+helm show values acp/kube-acp-stack
+```
+
+You may also `helm show values` on this chart's [dependencies](#dependencies) for additional options.
+
+### Multiple releases
+
+The same chart can be used to run multiple ACP instances in the same cluster if required. To achieve this, it is necessary to run only one instance of ACP, while all other components need to be disabled. To disable a dependency during installation, set `cockroachdb.enabled`, and `hazelcast.enabled` to `false`.
```

Comment:
Do you mean here multiple acp instances in different versions that are using the same crdb and hz? Or multiple pods?

Reply:
Multiple acp instances, but on second thought I think we should remove that section.